pkg/windows/windows.go

```diff
@@ -496,6 +496,12 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 	hybridOverlayServiceArgs := "--node " + nodeName + customVxlanPortArg + " --k8s-kubeconfig c:\\k\\kubeconfig " +
 		"--windows-service " + "--logfile " + hybridOverlayLogDir + "hybrid-overlay.log"
 
+	// check log level and increase hybrid-overlay verbosity if needed
+	if ctrl.Log.V(1).Enabled() {
+		// append loglevel param using 5 for debug (default: 4)
+		hybridOverlayServiceArgs = hybridOverlayServiceArgs + " --loglevel 5"
```

Comment:
if the default is 4 wouldnt raising it to 5 make it less verbose?

Reply:
Will add a reference link to the [code](https://github.com/openshift/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L736) for clarification.